nc_test4/Makefile.am

```diff
@@ -46,6 +46,15 @@ endif
 check_PROGRAMS = $(NC4_TESTS) tst_empty_vlen_unlim tst_charvlenbug
 TESTS = $(NC4_TESTS) run_empty_vlen_test.sh
 
+# This test calls MPI_Init if pnetcdf is turned on
+if USE_PNETCDF
+check_PROGRAMS += tst_atts3
+TESTS += run_atts3
+else
+TESTS += tst_atts3
+endif
+
```

Comment:
This is not correct because tst_atts3 should be run even if USE_PNETCDF is false.

Reply:
Exactly, and adding the program binary name to the TESTS variable will result in the program tst_atts3 being run without an MPI starter.
Or am I mistaken and the program calls MPI_Init based on some run-time decision?